FirebaseAuth/Tests/Sample/Podfile

```diff
@@ -17,6 +17,7 @@ target 'AuthSample' do
   pod 'FBSDKLoginKit', '< 13'
   pod 'GoogleSignIn', '~> 6'
   pod 'GTMSessionFetcher/Core'
+  pod 'FirebaseAppCheck'
```

Comment:
You can remove this from the PR as well.

Reply:
fixed all